pages/demo-game.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Link from "next/link";
+import { Container } from "react-bootstrap";
+
+export default function game() {
+    return (
+        <Container>
+            <h1>Hello, world!</h1>
+            <Link href="/about">About</Link>
+            <h1 class="text-centered">
+                ⬜⬜⬜⬜⬜ <br></br>
+                ⬜⬜⬜⬜⬜ <br></br>
+                ⬜⬜⬜⬜⬜ <br></br>
+                ⬜⬜⬜⬜⬜ <br></br>
+                ⬜⬜⬜⬜⬜ <br></br>
+            </h1>
+            <h2 class="text-centered">
```

Comment:
when we ran it, this was not actually centered in the screen. could we meet up soon to fix this possibly?